### PR TITLE
Custom commands

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: q.plugin.zsh [hunk 1]
```diff
@@ -7,6 +7,29 @@ ZSH_HIGHLIGHT_REGEXP+=('\bq.*\b' 'fg=green,bold')
 ZSH_HIGHLIGHT_REGEXP+=('\bQ.*\b' 'fg=green,bold')
 ZSH_HIGHLIGHT_REGEXP+=('\bU.*\b' 'fg=green,bold')
 
+# Check for a custom Q command
+echo $Q_SET
+echo $Q_RUN
+echo $Q_UNSET
```
Comment (cal2195) [inline q.plugin.zsh]: I think all these echo's should be gone? :)

**Review by cal2195 — CHANGES_REQUESTED** [review]

File: q.plugin.zsh [hunk 2]
```diff
@@ -7,6 +7,29 @@ ZSH_HIGHLIGHT_REGEXP+=('\bq.*\b' 'fg=green,bold')
...
+
+# Check if Q_SET is defined
+if [[ -z $Q_SET ]]; then
+    echo 'Using default set'
```
Comment (cal2195) [inline q.plugin.zsh]: And here! ;)

**Review by cal2195 — CHANGES_REQUESTED** [review]

File: q.plugin.zsh (hunk 2 above)
Comment (ghost) [reply q.plugin.zsh]: You want those ones gone too?

Comment (cal2195) [reply q.plugin.zsh]: Yeah! Don't need to tell the user everytime! ;)

**Review by cal2195 — APPROVED** [review]

**Pull request merged**